OWML.ModLoader/ModFinder.cs

```diff
@@ -42,6 +42,12 @@ private IModData GetModData(IModManifest manifest)
             var storage = new ModStorage(manifest);
             var config = storage.Load<ModConfig>(Constants.ModConfigFileName);
             var defaultConfig = storage.Load<ModConfig>(Constants.ModDefaultConfigFileName);
+            if (!manifest.RequireVR)//assume it wasn't present in manifest but present in config, to be removed
+            {
+                manifest.RequireVR = (config != null && config.RequireVR)
+                    || (config == null && defaultConfig != null && defaultConfig.RequireVR);
+                storage.Save(manifest, Constants.ModManifestFileName);
```

Comment:
Don't think OWML should ever edit a manifest file. Won't this work the same without this line anyway? Would just always set the internal manifest.RequireVR to whatever is in config, on every startup, but not save it to the file.

Reply:
Hmmm... I guess you're correct. It's only intended to persist until 0.5.2 anyway